src/script/src/python/coprocessor/compile.rs

```diff
@@ -31,23 +31,40 @@ fn create_located<T>(node: T, loc: Location) -> Located<T> {
 /// generate a call to the coprocessor function
 /// with arguments given in decorator's `args` list
 /// also set in location in source code to `loc`
-fn gen_call(name: &str, deco_args: &DecoratorArgs, loc: &Location) -> ast::Stmt<()> {
```

Comment:
Just realize we can just directly use `vm.call_method(obj, method_name, args)` to call methods with `FuncArgs`(Support PosArg&KwArg) manually constructed. Should we change it here or create a new PR?

Reply:
I can try it in this PR.